var/spack/repos/builtin/packages/sensei/package.py

```diff
@@ -98,7 +143,17 @@ def cmake_args(self):
             self.define_from_variant('ENABLE_OSCILLATORS', 'miniapps')
         ]
 
+        if '+adios2' in spec:
+            args.append(
+                self.define('ADIOS2_DIR', spec['adios2'].prefix))
+
+        if '+ascent' in spec:
+            args.append(
+                self.define('ASCENT_DIR', spec['ascent'].prefix))
+
         if '+libsim' in spec:
+            # This is only for linux
+            # Visit install location may be different on other platforms
             args.append(
                 '-DVISIT_DIR:PATH={0}/current/{1}-{2}'.format(
                     spec['visit'].prefix, spec.platform, spec.target.family)
```

Comment:
This needs a patch as well to work with visit.

`'-DVISIT_DIR:PATH={0}/current/linux-x86_64'.format(spec['visit'].prefix)`